assets/src/BlockEditorValidation.js

```diff
@@ -26,16 +32,16 @@ export const blockEditorValidation = () => {
 
         return results;
       }, [] );
-      if (results.length > 0) {
-        invalidBlocks.push(...results)
-      }
+
+      invalidBlocks.push(...results);
```

Comment:
I didn't intend to make this change but it was a compromise with Sonarcloud to get back down to the allowed complexity...

Reply:
Sonar is completely right btw that this function is too complex, but I'm avoiding the context switch atm to dive into this logic. I might address it later.